### PR TITLE
Add cfs test

### PR DESCRIPTION
**Pull request by gnanirahulnutakki**
Initial PR for cfs helm chart

### CODE REVIEW

File: charts/cfs/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
Comment (pgodey) [inline charts/cfs/Chart.yaml:24]: This should match the default tag name.

File: charts/cfs/values.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+replicaCount: 1
+
+image:
+  repository: radiantone/cfs-master
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "3.16.1-ltsc2019"
```
Comment (pgodey) [inline charts/cfs/values.yaml:7]: This should be blank.

**Review by pgodey — COMMENTED** [review]
Remove all .DS_Store files from the repo

File: charts/cfs/Chart.yaml (hunk 1 above)
Comment (gnanirahulnutakki) [reply charts/cfs/Chart.yaml:24]: Changed appVersion to default tag name

File: charts/cfs/values.yaml (hunk 2 above)
Comment (gnanirahulnutakki) [reply charts/cfs/values.yaml]: Changed to blank

**Comment by gnanirahulnutakki** [comment]
Made the changes.

**Pull request merged**